reporter/src/main/kotlin/reporters/TableReporter.kt

```diff
@@ -208,11 +229,11 @@ abstract class TableReporter : Reporter() {
                 }
             }
 
-            Pair(project, ProjectTable(tableRows))
+            Pair(project, ProjectTable(tableRows, projectExclude))
         }.toSortedMap()
 
         val errorSummaryTable = ErrorTable(errorSummaryRows.values.toList().sortedBy { it.id })
-        val summaryTable = ProjectTable(summaryRows.values.toList().sortedBy { it.id })
+        val summaryTable = ProjectTable(summaryRows.values.toList().sortedBy { it.id }, null)
```

Comment:
Would it be better if that magic number (null) was eliminated?

Reply:
The first param isn't obvious either, maybe use named parameters for both!

Reply:
Made `null` the default value for the second argument, so this line is not changed anymore.